src/edit/index.vue

```diff
@@ -82,11 +91,18 @@ export default {
             fileName: null
           });
         });
+    },
+    reload() {
+      const { playable } = this.element.data;
+      if (playable) this.$emit('save', this.element.data);
```

Comment:
Why not simply reload the `iframe` through `$refs`?

Reply:
Good catch! This is a mistake caused by switching the context between API video and Sprout video 😅 .